PrivateHospitals/backend/PrivateHospitals/PrivateHospital.Migration/Interfaces/IRepository.cs

```diff
@@ -0,0 +1,7 @@
+namespace PrivateHospital.Migration.Dto.Interfaces;
+
+public interface IRepository<T> where T: class
+{
+    Task<T> GetByExternalId(string externalId);
```

Comment:
Why would u locate it here, not every entity has external id, but u enforce the implementatioon